packages/sdk-themes/src/index.ts

```diff
@@ -1,10 +1,12 @@
 export {
-  moniteLight,
+  defaultMoniteComponents,
```

Comment:
I realized the problem now. We cannot afford to customize the main theme because it will be difficult to support.
Developers could start using the custom values in the main SDK. Then, we will have to make customers describe not only the standard MUI Theme Options but also custom variables. So, we have to avoid using custom variables in SDK and we shouldn't show and promote it for now.

We should keep the following theme options splitting:
`sdk-themes` should contain common styles only.
Project-specific styles should exist within the project.

There are more limitations. Some of them are not so easy to comprehend. I suggest we should discuss it in Slack, I will start a conversation today.